ui/src/chronograf/components/Panel.js

```diff
@@ -40,16 +41,26 @@ const Explorer = React.createClass({
     this.setState({activeQueryId: query.id});
   },
 
+  toggleAddQueryOptions() {
+    this.setState({showAddQueryOptions: !this.state.showAddQueryOptions});
+  },
+
   handleAddQuery() {
     this.props.actions.addQuery();
+    this.setState({showAddQueryOptions: false});
+  },
+
+  handleAddRawQuery() {
+    this.props.actions.addQuery({rawText: `Select "field" from "measurement"`});
+    this.setState({showAddQueryOptions: false});
```

Comment:
Super pedantic and I know it doesn't really matter because this is placeholder text BUT this should probably read:

`SELECT "fields" FROM "db"."rp"."measurement"`